core/rawdb/accessors_chain.go

```diff
@@ -752,8 +751,15 @@ func MakeBodiesCanonical(tx kv.RwTx, from uint64, ctx context.Context, logPrefix
 		i := uint64(0)
 		if err := tx.ForAmount(kv.NonCanonicalTxs, hexutility.EncodeTs(bodyForStorage.BaseTxId+1), bodyForStorage.TxAmount-2, func(k, v []byte) error {
 			id := newBaseId + 1 + i
-			if err := tx.Put(kv.EthTx, hexutility.EncodeTs(id), v); err != nil {
-				return err
+			if transactionsV3 {
+				key := append(hexutility.EncodeTs(id), h.Bytes()...)
+				if err := tx.Put(kv.EthTxV2, key, v); err != nil {
```

Comment:
Let’s rename to V3 to align with ErigonV3

Reply:
Yeah! Thanks, good point! will do.